styles.css

```diff
@@ -363,8 +335,255 @@ p.paragraph {
 .dev-email {
   color: #42526e;
   text-decoration: none;
-  margin-top: 0.8rem;
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
+  align-items: center;
+}
+
+.mywork-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.socials2 {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 0.8rem;
+}
+
+.desktop-nav-container {
+  visibility: hidden;
+}
+
+.background-desktop {
+  background-image: url(./tools/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: initial;
+  width: 100%;
+}
+
+@media all and (min-width: 768px) {
+  .welcome {
+    width: 100%;
+    padding: 0;
+    bottom: 100px;
+    margin-top: 8rem;
+    margin-bottom: 25rem;
+  }
+
+  .background-desktop {
+    background-image: url(./tools/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: initial;
+    width: 100%;
+  }
+
+  .description {
+    text-align: center;
+    width: 920px;
+    height: 96px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .body {
+    flex: none;
+    width: 100%;
+  }
+
+  .socials {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+    gap: 2.1rem;
+    justify-content: center;
+  }
+
+  .mywork {
+    width: 100%;
+    padding: 0;
+  }
+
+  .recents-page {
+    justify-content: center;
+  }
```

Comment:
A look at the hero page reveals that there's more margin between the background image and the recent-works heading than you have represented. Kindly adjust this if you may, to match the design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199930631-738968ca-cdda-4952-81bd-c07f76b8a618.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199930531-dd35a501-a81f-4914-828b-700690edad22.png)